tee-worker/omni-executor/client-sdk/packages/client-sdk/src/lib/requests/__test__/pumpx-add-export-wallet.request.test.ts

```diff
@@ -0,0 +1,61 @@
+import { enclave } from '@lib/enclave';
+import { PumpxRpcMethod } from '@lib/utils/rpc-method';
+import { JsonRpcRequest } from '@utils/types';
+
+// This test is just an example. It requires receiving email verification codes, which cannot be done by running this unit test.
+describe.skip('add-export-wallet', () => {
+  // step 1
+  it('request email verification code', async () => {
+    const rpcRequest = {
+      jsonrpc: '2.0',
+      method: 'omni_requestEmailVerificationCode',
+      params: ['test@google.com'],
+    };
+    await enclave.send(rpcRequest);
+  });
+
+  // step 2
+  it('request email verification code', async () => {
+    const rpcRequest: JsonRpcRequest = {
+      jsonrpc: '2.0',
+      method: 'pumpx_requestJwt',
+      params: ['test@google.com', '', '', '', '1223'],
+    };
+
+    const response = await enclave.send(rpcRequest);
+    // console.log(response.auth_token);
+  });
+  it('add wallet should works', async () => {
+    const rpcRequest: JsonRpcRequest = {
+      jsonrpc: '2.0',
+      method: PumpxRpcMethod.AddWallet,
+      id: 1,
+      params: [
+        'test@google.com', // user_email
+        '0xxxxxx', // auth_token already get from step 2
```

Comment:
We are changing all pumpx rpc methods to use an object as params so this and requests will need to be adjusted

Reply:
Thanks for heads up.I will check all the files.